src/url/dto/create-url-request-data.ts

```diff
@@ -1,12 +1,18 @@
-import { IsNotEmpty } from 'class-validator';
+import { Type } from 'class-transformer';
+import { IsDate, IsNotEmpty, IsString, IsUrl } from 'class-validator';
 
 export class CreateUrlRequestData {
-  @IsNotEmpty({ message: 'Expiry date cannot be empty' })
-  expiresAt: Date;
+  @IsNotEmpty()
+  @IsDate()
+  @Type(() => Date)
+  readonly expiresAt?: Date;
```

Comment:
if the field is `@IsNotEmpty()` would it be suitable to add "?" after `expiredAt`
```suggestion
  readonly expiresAt: Date;
```

Reply:
updated